federated_content_connector/management/commands/tests/mock_responses.py

```diff
@@ -0,0 +1,361 @@
+                    "is_marketable": True,
+                    "availability": "Current",
+                    "course": "edX+DemoX",
+                    "full_description": "This is CS50x , Harvard University's introduction to the ia",
+                    "announcement": None,
+                    "video": None,
+                    "content_language": None,
+                    "license": "",
+                    "outcome": "",
+                    "transcript_languages": [],
+                    "instructors": [],
+                    "staff": [],
+                    "min_effort": None,
+                    "max_effort": None,
+                    "modified": "2023-05-22T18:02:32.723407Z",
+                    "level_type": None,
+                    "mobile_available": False,
+                    "hidden": False,
+                    "reporting_type": "mooc",
+                    "eligible_for_financial_aid": True,
+                    "first_enrollable_paid_seat_price": 149,
+                    "has_ofac_restrictions": None,
+                    "ofac_comment": "",
+                    "enrollment_count": 0,
+                    "recent_enrollment_count": 0,
+                    "expected_program_type": None,
+                    "expected_program_name": "",
+                    "course_uuid": "102c166f-af2b-4fe2-ade0-ab08063d4d40",
+                    "estimated_hours": 0,
+                    "content_language_search_facet_name": None,
+                    "enterprise_subscription_inclusion": False
+                }
+            ],
+            "entitlements": [
+                {
+                    "mode": "verified",
+                    "price": "149.00",
+                    "currency": "USD",
+                    "sku": "9023CB5",
+                    "expires": None
+                }
+            ],
+            "owners": [
+                {
+                    "uuid": "2906c944-5698-4e5d-bc18-a2bfe8d343af",
+                    "key": "edX",
+                    "name": "edX",
+                    "auto_generate_course_run_keys": True,
+                    "certificate_logo_image_url": None,
+                    "logo_image_url": None,
+                    "organization_hex_color": None,
+                    "description": None,
+                    "description_es": "",
+                    "homepage_url": None,
+                    "tags": [],
+                    "marketing_url": "school/edx",
+                    "slug": "edx",
+                    "banner_image_url": None,
+                    "enterprise_subscription_inclusion": False
+                }
+            ],
+            "image": None,
+            "short_description": "An introduction to the intellectual enterprises of computer science.",
+            "type": "97423c19-8254-43b6-bb7e-33b1f3efe246",
+            "url_slug": "introduction-to-computer-science",
+            "course_type": "executive-education-2u",
```

Comment:
If this course is for an exec ed course, should the metadata more closely reflect how an exec ed course is represented on stage/prod? That is, there is 1 course run with a single seat type of `unpaid-executive-education`. The `entitlements` contains a type of `paid-executive-education`.

Example: https://discovery.edx.org/api/v1/courses/UCT+SCM/

Reply:
Great Catch. Updated the mock response.